src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+}
+
+.moyo__logo {
+  margin-top: 5px;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Consider adding a generic font family as a fallback after 'Roboto', such as 'sans-serif'.